core/src/main/java/io/questdb/cutlass/pgwire/DefaultPgWireAuthenticationFactory.java

```diff
@@ -24,11 +24,19 @@
 
 package io.questdb.cutlass.pgwire;
 
+import io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker;
+import io.questdb.cutlass.auth.Authenticator;
+import io.questdb.network.NetworkFacade;
+
 public class DefaultPgWireAuthenticationFactory implements PgWireAuthenticationFactory {
     public static final PgWireAuthenticationFactory INSTANCE = new DefaultPgWireAuthenticationFactory();
 
     @Override
-    public ClearTextPgWireAuthenticator getPgWireAuthenticator(PGConnectionContext.ResponseAsciiSink responseAsciiSink, PGWireConfiguration configuration) {
-        return new ClearTextPgWireAuthenticator(responseAsciiSink, configuration);
+    public Authenticator getPgWireAuthenticator(NetworkFacade nf,
+                                                PGWireConfiguration configuration,
+                                                NetworkSqlExecutionCircuitBreaker circuitBreaker,
+                                                CircuitBreakerRegistry registry,
+                                                OptionsListener optionsListener) {
+        return new CleartextPasswordPgWireAuthenticator(nf, configuration, circuitBreaker, registry, optionsListener, new StaticUserDatabase(configuration));
```

Comment:
there is no need to create static database more than once

Reply:
@bluestreak01 looks like this is the only non-resolved issue. I know you were planning to look at this last night. any success? do you want me to look at this? FYI: it's only relevant with the DefaultServerConfiguration, not when property-based config is used.